frb_spectral_search/spectral_search.py

```diff
@@ -980,4 +979,4 @@ def align_yaxis(ax1, ax2):
     b_new_t = extrema[0][0] + tot_span * (extrema[0][1] - extrema[0][0])
```

Comment:
I don't really understand what `align_yaxis` is supposed to do, but might `sharey=True` be helpful here?